rust-connectors/sources/http/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Connector Change Log
 
-## http Version 0.2.1 - UNRELEASED
+## http Version 0.3.0 - UNRELEASED
+* `interval` field uses human readable values, "1s", "20ms", etc. ([PR #274](https://github.com/infinyon/fluvio-connectors/pull/274))
```

Comment:
Also we can mention about linger time support

Reply:
That's an option shared amongst all the connectors. We need to figure out a good way to show all the common options.